lib/SIL/IR/OperandOwnership.cpp

```diff
@@ -305,7 +360,7 @@ ACCEPTS_ANY_OWNERSHIP_INST(ConvertEscapeToNoEscape)
 #include "swift/AST/ReferenceStorage.def"
 #undef ACCEPTS_ANY_OWNERSHIP_INST
 
-OperandOwnershipKindMap
+RawConstraintInfo
```

Comment:
NOTE: Change this code to use the ValueOwnershipKind stored in the forwarding inst.